noxfile-template.py

```diff
@@ -133,7 +133,10 @@ def _determine_local_import_names(start_dir):
 
 @nox.session
 def lint(session):
-    session.install("flake8", "flake8-import-order")
+    if not TEST_CONFIG['enforce_type_hints']:
```

Comment:
Can you also add `enforce_type_hints` in the `TEST_CONFIG` above in this file?

Reply:
thank you!! good catch. It's in the template template, I just forgot it here.